Bond.cc

```diff
@@ -10,8 +10,9 @@ Bond::Bond() {}
 
 Bond::~Bond() {}
 
-void Bond::setEODStockPrice() {}
-float Bond::getEODReturns() const {return 0.0;}
+float Bond::getEODReturns() const {
+  return 0.0;
```

Comment:
Should return `dividendsPerShare` here. Also, can you rename this function to `getEODReturnsPerShare()` (remeber to change for BasicStock as well)

Reply:
Inside the game manager we can then do something like this `numShares * stock.getEODReturnsPerShare()`

Reply:
I was thinking of returning `dividendsPerShare`, but then I saw your function called `getDividendPerShare`, which does exactly the same thing so I didn't do it. Are you sure you want to duplicate this functionality? We can just use `getDividendPerShare` and remove `getEODReturns` altogether from `BasicBond` class, which returns 0 anyway, and the `Bond` class.

Reply:
remove the getDividendPerShare function instead. I want to be able to get all the returns without having to know if a stock is a bond or basicStock instance